lib/JsonpMainTemplatePlugin.js

```diff
@@ -36,7 +36,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"var script = document.createElement('script');",
 			"script.type = 'text/javascript';",
 			"script.charset = 'utf-8';",
-			"script.async = true;",
+			"script.async = false;",
```

Comment:
Noticed that modules, wrapped to functions are executed by IE if they gets from cache
`(function(){
   //code
})()`

chunks are coming like
`window.someFunc()`
and don't execute in that case